src/mem/freelist.h

```diff
@@ -383,62 +298,29 @@ namespace snmalloc
         // If second list is non-empty, perform append.
         if (end[1] != &head[1])
         {
+          // The start token has been corrupted.
+          // TOUTOC issue, but small window here.
```

Comment:
```suggestion
          // TOCTTOU issue, but small window here.
```
is the more traditional spelling, I think?